src/abbfreeathome/devices/blind_sensor.py

```diff
@@ -46,14 +58,19 @@ def __init__(
         )
 
     @property
-    def state(self) -> bool | None:
+    def state(self) -> str | None:
         """Get the sensor state."""
-        return self._state
+        return self._state.name
 
     @property
-    def longpress(self) -> bool | None:
-        """Get the info, if the rocker was long pressed."""
-        return self._longpress
+    def step_state(self) -> BlindSensorState:
```

Comment:
Shouldn't this also return a `str`, like the `state`?

Reply:
It could be, either way shouldn't really matter. Home Assistant will use the `state` property. But if we want to be consistant across all properties I can switch to string.

Reply:
Yep, I would vote for string to be consistant. Nothing more